plugins/kubernetes/app/assets/stylesheets/kubernetes/application.css

```diff
@@ -1,87 +1,28 @@
-@-ms-keyframes spin {
-  from {
-    -ms-transform: rotate(0deg);
-  }
-  to {
-    -ms-transform: rotate(360deg);
-  }
-}
-
-@-moz-keyframes spin {
-  from {
-    -moz-transform: rotate(0deg);
-  }
-  to {
-    -moz-transform: rotate(360deg);
-  }
-}
 
-@-webkit-keyframes spin {
-  from {
-    -webkit-transform: rotate(0deg);
-  }
-  to {
-    -webkit-transform: rotate(360deg);
-  }
+/*
+    Kubernetes Releases
+*/
+us-spinner.kubernetes-releases-spinner {
+  padding: 100px 0;
 }
 
-@keyframes spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
-}
-
-.spinning {
-  -webkit-animation-name: spin;
-  -webkit-animation-duration: 4000ms;
-  -webkit-animation-iteration-count: infinite;
-  -webkit-animation-timing-function: linear;
-  -moz-animation-name: spin;
-  -moz-animation-duration: 4000ms;
-  -moz-animation-iteration-count: infinite;
-  -moz-animation-timing-function: linear;
-  -ms-animation-name: spin;
-  -ms-animation-duration: 4000ms;
-  -ms-animation-iteration-count: infinite;
-  -ms-animation-timing-function: linear;
-  animation-name: spin;
-  animation-duration: 4000ms;
-  animation-iteration-count: infinite;
-  animation-timing-function: linear;
-}
-
-.spinning.ng-animate {
-  -webkit-animation: none;
-  -moz-animation: none;
-  -ms-animation: none;
-  animation: none;
-}
-
-.kubernetes .md-tab {
-  color: #358cce;
-  text-transform: none;
+us-spinner.kubernetes-releases-spinner [role="progressbar"]::after {
+  content: "Loading Kubernetes Releases";
 }
 
-.kubernetes .md-tab.md-active {
-  color: rgb(255, 205, 210);
+.kubernetes-release-groups span.label-info {
+  margin: 0 5px;
 }
 
-.kubernetes [role="progressbar"] {
-  transform: scale(0.4) !important;
-  margin: 50px auto !important;
+/*
+  Kubernetes Roles
+*/
+us-spinner.kubernetes-roles-spinner {
+  padding: 100px 0;
 }
 
-.kubernetes [role="progressbar"]::after {
-  position: absolute;
-  font-size: 14px;
-  transform: scale(2) !important;
-  width: 150px;
-  text-align: center;
-  top: 140px;
-  left: -20px;
+us-spinner.kubernetes-roles-spinner [role="progressbar"]::after {
+  content: "Loading Kubernetes Roles";
```

Comment:
A bit weird to put non-styling content here, Were you not able to put this text between ```us-spinner``` tags?